src/Dfe.PlanTech.Web/ViewBuilders/CategorySectionViewComponentViewBuilder.cs

```diff
@@ -53,7 +53,7 @@ public async Task<CategorySectionViewComponentViewModel> BuildViewModelAsync(Que
             CategoryHeaderText = category.Header.Text,
             CategorySections = categorySections,
             CategorySlug = categoryLandingSlug,
-            CompletedSectionCount = sectionStatuses.Count(ss => ss.Completed),
+            CompletedSectionCount = sectionStatuses.Count(ss => ss.LastCompletionDate.HasValue),
```

Comment:
There seems to be at least one similar example of this logic/filter:

`Dfe.PlanTech.Web.ViewBuilders.CategorySectionViewComponentViewBuilder.BuildViewModelAsync`

```csharp
        return new CategorySectionViewComponentViewModel
        {
            CategoryHeaderText = category.Header.Text,
            CategorySections = categorySections,
            CategorySlug = categoryLandingSlug,
            CompletedSectionCount = sectionStatuses.Count(ss => ss.Completed), // does this need to be updated too?
            Description = description,
            ProgressRetrievalErrorMessage = progressRetrievalErrorMessage,
            TotalSectionCount = category.Sections.Count
        };
```